### PR TITLE
updated performance docs

### LINKED ISSUE #3214: [Docs] Update Performance usage example
#### Current State
image

#### Desired State.
Update the pages https://docs.meshery.io/reference/mesheryctl and https://docs.meshery.io/reference/mesheryctl/perf with the latest changes in `mesheryctl perf` command.

---

#### Contributor Resources

- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by NagaHarshita**
Signed-off-by: NagaHarshita <harshitamarupaka@gmail.com>

**Description**

This PR fixes #3214.
The output of the help command is taken as a reference to update the docs. Please suggest changes if any.

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thank you, @NagaHarshita. Good work. 😄 

After giving some thought as to how we describe the `--token` flag, do mind changing all references to use the phrase "access token", instead of "auth config"?

File: docs/_data/mesheryctlcommands/performance/view/flag/output-format.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+flag: --output-format, -o
+arg: view
+description: (optional) format to display in json
```
[the remark this answers is not in the document]
Comment (piyushsingariya) [reply docs/_data/mesheryctlcommands/performance/view/flag/output-format.yaml]: We support both YAM/JSON, few things need to be updated in cobra documentation for `perf` commands.

**Review by piyushsingariya — CHANGES_REQUESTED** [review]
@NagaHarshita The flags are under `apply` subcommand under `perf`. plus `perf list` is also not added.

**Comment by leecalcote** [comment]
Good progress, @NagaHarshita 👍  Perhaps, we should breakdown this work into multiple PRs to make it a more digestible set of changes?

File: docs/_data/mesheryctlcommands/performance/view/flag/output-format.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (leecalcote) [reply docs/_data/mesheryctlcommands/performance/view/flag/output-format.yaml]: @piyushsingariya are you tracking an issue to update the help text for this command?

**Comment by leecalcote** [comment]
> @NagaHarshita The flags are under `apply` subcommand under `perf`. plus `perf list` is also not added.

Good catch @piyushsingariya. @NagaHarshita, do you feel that you can address these subcommands in this PR or is opening up a new PR to tie off the rest of the subcommands easier? If the latter, please open up an issue to track the need for `apply` and `list`

**Comment by NagaHarshita** [comment]
> Good catch @piyushsingariya. @NagaHarshita, do you feel that you can address these subcommands in this PR or is opening up a new PR to tie off the rest of the subcommands easier? If the latter, please open up an issue to track the need for `apply` and `list`
I will try to complete the mentioned changes in this pr.

**Review by leecalcote — APPROVED** [review]
Looking good, @NagaHarshita. 😀

@piyushsingariya, what do you think?

**Review by piyushsingariya — CHANGES_REQUESTED** [review]
@NagaHarshita Can you replace all the profile-id under `perf apply` with profile-name. With some recent changes we have shifted the `perf apply` command to work with names instead of ids.
Keep in mind `list` and `view` work same as before.

**Pull request merged**